osu.Game.Rulesets.Taiko/Difficulty/TaikoDifficultyCalculator.cs

```diff
@@ -116,16 +119,23 @@ protected override DifficultyAttributes CreateDifficultyAttributes(IBeatmap beat
             double monoStaminaFactor = staminaRating == 0 ? 1 : Math.Pow(monoStaminaRating / staminaRating, 5);
 
             double colourDifficultStrains = colour.CountTopWeightedStrains();
-            double readingDifficultStrains = reading.CountTopWeightedStrains();
-            double staminaDifficultStrains = stamina.CountTopWeightedStrains();
+            double rhythmDifficultStrains = rhythm.CountTopWeightedStrains();
+            double staminaDifficultStrains = stamina.CountTopWeightedStrains() * clockRate;
```

Comment:
Why is it multiplied by clockrate here? Shouldn't clockrate be included in the strain calculation already?